lib/Froxlor/Cron/System/TasksCron.php

```diff
@@ -257,11 +258,18 @@ private static function rebuildDnsConfigs()
 
 		$nameserver = new $dnssrv(\Froxlor\FroxlorLogger::getInstanceOf());
 
+		$nameserver->writeConfigs();
+	}
+
+	private static function rebuildDKIMConfigs()
+	{
+		$dnssrv = '\\Froxlor\\Cron\\Dns\\' . Settings::Get('system.dns_server');
```

Comment:
why create instance of dns-class and later only run writeDKIMconfigs() only if enabled? Best is to wrap the whole function body in a `if (Settings::Get('dkim.use_dkim') == '1') { }`-block